### PR TITLE
Template pipeline.conf files in chef

### PR DESCRIPTION
**Pull request by anguss00**
Fixes https://github.com/aodn/zzz-aodn-pipeline-poc/issues/103

### CODE REVIEW

**Comment by anguss00** [comment]
Fixed @lwgordonimos

File: private-sample/nodes/po_pipeline_2.json [hunk 1]
```diff
@@ -59,6 +59,45 @@
...
+            "sender": "info@aodn.org.au",
+            "smtp_server": "email-smtp.us-east-1.amazonaws.com"
+        },
+        "pipeline": {
+            "config": {
+                "global": {
+                    "admin_recipients": "email:leigh.gordon@utas.edu.au,email:angus.scheibner@utas.edu.au",
+                    "upload_uri": "file:///s3/imos-data",
+                    "wip_dir": "/tmp/wip",
+                    "error_dir": "/mnt/ebs/error"
```
Comment (mhidas) [inline private-sample/nodes/po_pipeline_2.json]: Missing "archive_uri" and "processing_dir"

Comment (anguss00) [reply private-sample/nodes/po_pipeline_2.json]: No worries, I'll amend

File: private-sample/nodes/po_pipeline_2.json [hunk 2]
```diff
@@ -59,6 +59,45 @@
...
+                "mail": {
+                    "from": "info@aodn.org.au",
+                    "subject": "AODN Pipeline Processing Report: {input_file}",
+                    "smtp_server": "email-smtp.us-east-1.amazonaws.com",
+                    "smtp_user": "ARRY",
+                    "smtp_pass": "BROWN"
+                },
+                "pluggable": {
+                    "handlers_group": "pipeline.handlers",
+                    "dest_path_function_group": "pipeline.dest_path_functions"
```
Comment (mhidas) [inline private-sample/nodes/po_pipeline_2.json]: "dest_path_function_group" has been renamed to "path_function_group"

File: private-sample/nodes/po_pipeline_2.json (hunk 1 above)
Comment (anguss00) [reply private-sample/nodes/po_pipeline_2.json]: what should they be by default in the PO box?

Comment (mhidas) [reply private-sample/nodes/po_pipeline_2.json]: I've used:
```
    "archive_uri": "file:///tmp/archive",
    "processing_dir": "file:///tmp/processing",
```

**Comment by anguss00** [comment]
done @mhidas

**Comment by mhidas** [comment]
Thanks. I'll let @lwgordonimos merge this. It did run up fine for me.

**Pull request merged**